doc/installation_windows.md

```diff
@@ -2,52 +2,89 @@
 
 Guide for Windows users, without touching the Command Line.
 
-## Install requirements
+## First steps
 
-**Python**
+1. [Register](https://github.com/join) on GitHub
+2. [Get](https://docs.github.com/en/enterprise-cloud@latest/desktop/installing-and-authenticating-to-github-desktop/installing-github-desktop#downloading-and-installing-github-desktop) `Github Desktop` [^1]
+3. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) Tileset repository
+4. [Clone](https://docs.github.com/en/desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop) forked repository.
 
-Install Python 3 from [python.org](https://www.python.org/downloads/windows/).
+> [!NOTE]
+> It is a good practice to create new branch at the start. Name it "playground" for example.
+> Your repo is your own private space. You can do whatever you want in it without affecting the original project. However, if you want to contribute to the project, you need to follow some guidelines. One of them is to keep your master branch clean and empty, and do your work in a separate branch. This will make it easier for you to sync your changes with the upstream repo and create pull requests.
 
-During installation, check "Add Python to PATH".
+Now you have source files on your drive and some tools as well.  
+These tools will help you managing multitile objects, setting up the environment and composing tilesets.  
 
-**libvips**
+## The Game
 
-Download the latest libvips distribution from [libvips.github.io](https://libvips.github.io/libvips/install.html)
-(get vips-dev-w64-web-#.#.#.zip NOT vips-dev-w64-all-#.#.#.zip).
```

Comment:
Maybe don't remove the infos about the correct version to get

Reply:
it is here, but a bit lower (line 69 in the next commit)